src/index.js

```diff
@@ -56,7 +59,10 @@ export default class EHoldings extends Component {
     } = this.props;
 
     return showSettings ? (
-      <Route path={rootPath} component={SettingsRoute} />
+      <Route path={rootPath} component={SettingsRoute}>
+        <Route path="knowledge-base" exact component={SettingsKnowledgeBaseRoute} name="Knowledge base" />
+        <Route path="root-proxy" exact component={SettingsRootProxyRoute} name="Root proxy" />
+      </Route>
```

Comment:
I like this much better. We should add a comment above the knowledge base route that says something like "these paths are not used by the Route component, but by the settings component later on"

Just because my immediate reaction to this is "these paths look wrong"

Reply:
Also, since we kinda just throw away these `Route` components after pulling the data off of them in `SettingsRoute`, `exact` is probably unnecessary. Although I could see the argument of keeping it if the upstream `Settings` component ends up supporting nested routes like this.

Reply:
I left `exact` so it's a little closer to the ideal. Comments added.